web/src/components/metrics/latency-donut.tsx

```diff
@@ -41,6 +41,7 @@ export const LatencyDonut: React.FC<LatencyDonutProps> = ({
       name: (m as NamedMetric).fullName || (m as GenericMetric).name,
       value: getStat(m.stats, 'avg')
     }))
+    .filter(m => !othersName || m.name !== othersName)
```

Comment:
can u pls explain what this change for ?

Reply:
Previously I was querying response codes different from `NoError` and finding back the `NoError` count from totals.
```
|~`"DnsFlagsResponseCode"`!~`"DnsFlagsResponseCode":"NoError"`
```

This has been [changed to all response codes](https://github.com/netobserv/network-observability-console-plugin/pull/425/files#diff-cbb81d7aa1abb66e2d7be256ed86f21544a0116e8ba5c5f0022852d5814cbcc4) for consistency:
```
|~`"DnsFlagsResponseCode"`
```
so the filtering needs to be done on client side when `showOthers` is not true.